st2common/bin/bootstraprmq.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python2.7
+
+import sys
+import st2common.transport.bootstrap as transport_bootstrap
+
+if __name__ == '__main__':
```

Comment:
Well I dislike that as well.

This way all processes don't end up bootstrapping RMQ. Also, there is a degree of control and can be done from outside.